Index: zhcon-0.2.6/src/display/lrmi.c
===================================================================
--- zhcon-0.2.6.orig/src/display/lrmi.c	2009-12-24 22:05:43.000000000 -0500
+++ zhcon-0.2.6/src/display/lrmi.c	2009-12-24 22:06:35.000000000 -0500
@@ -33,6 +33,13 @@
 
 #include "lrmi.h"
 
+#if defined(__linux__) && !defined(TF_MASK)
+	#define TF_MASK X86_EFLAGS_TF
+	#define IF_MASK X86_EFLAGS_IF
+	#define VIF_MASK X86_EFLAGS_VIF
+	#define IOPL_MASK X86_EFLAGS_IOPL
+#endif
+
 #define REAL_MEM_BASE 	((void *)0x10000)
 #define REAL_MEM_SIZE 	0x10000
 #define REAL_MEM_BLOCKS 	0x100
